src/ProductConstructionService/ProductConstructionService.DependencyFlow/NonBatchedPullRequestUpdater.cs

```diff
@@ -66,8 +66,12 @@ public NonBatchedPullRequestUpdater(
         Subscription? subscription = await _context.Subscriptions.FindAsync(SubscriptionId);
         if (subscription == null)
         {
-            await _pullRequestCheckReminders.UnsetReminderAsync();
-            await _pullRequestUpdateReminders.UnsetReminderAsync();
+            // We don't know if the subscription was a code flow one, so just uncheck both
```

Comment:
We do know - `subscription.SourceEnabled`

Reply:
Technically this is maybe something we could bind at the manager creation time and not have to supply the boolean?